extensions/scarb-execute/src/lib.rs

```diff
@@ -251,6 +261,59 @@ pub fn execute(
         fs::write(air_private_input_path, output_value)?;
     }
 
+    if args.run.save_profiler_trace_data {
+        ensure!(
+            package.targets.iter().any(|t| t.kind == "executable"
+                && t.params.get("sierra").and_then(|v| v.as_bool()) == Some(true)),
+            "Failed to write profiler trace data into a file - missing sierra code. \
+            Set `sierra = true` under your `[executable]` target in the config and try again."
+        );
+        let profiler_trace_path = Utf8PathBuf::from(format!(
+            "{}/{}/{}.executable.sierra.json",
+            &metadata.workspace.root, &scarb_build_dir, &build_target.name
```

Comment:
scarb_build_dir should be enough, it is an absolute path.

```suggestion
            &scarb_build_dir, &build_target.name
```